.github/workflows/deploy-clojars.yml

```diff
@@ -0,0 +1,31 @@
+name: deploy-clojars
+on: [ workflow_dispatch ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: clojure:openjdk-11-tools-deps
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: clj -Adepstar
+      - uses: actions/upload-artifact@v2
+        with:
+          name: jar
+          path: cruler.jar
+
+  deploy:
+    needs: build-jar
```

Comment:
`build-jar` is not found. typo of `build`?

Reply:
Oh thx. fix https://github.com/xcoo/cruler/pull/7/commits/00af681e3e1fd3e92986513867c428692f59e289